File: Ent_UsAdmin/Ent_UsAdmin/CatTiposdeCredito.vb
Imports System.Data.OleDb
Imports System.Data.SqlClient

Public Class CatTiposdeCredito
    Private Sub CatTiposdeCredito_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        cargarTipos()
    End Sub
    Public Sub cargarTipos()
        dtimpuestos.Rows.Clear()
        Try
            Dim strimpuestos As String
            iniciarconexionempresa()

            strimpuestos = "select * from TiposDeCredito"

            Dim ejec = New SqlCommand(strimpuestos)
            ejec.Connection = conexionempresa
            Dim id, nombre, Modalidad, Tipo, Plazo As String

            Dim myreaderimpuestos As SqlDataReader = ejec.ExecuteReader()
            While myreaderimpuestos.Read
                id = myreaderimpuestos("id")
                nombre = myreaderimpuestos("nombre")
                Modalidad = myreaderimpuestos("Modalidad")
                Tipo = myreaderimpuestos("Tipo")
                Plazo = myreaderimpuestos("Plazo")
                dtimpuestos.Rows.Add(id, nombre, Modalidad, Tipo, Plazo)

            End While
            myreaderimpuestos.Close()
        Catch ex As Exception
            MessageBox.Show(ex.Message)
        End Try

    End Sub

    Private Sub btn_agregar_Click(sender As Object, e As EventArgs) Handles btn_agregar.Click
        AgregarTipoCredito.Show()
    End Sub

    Private Sub BunifuThinButton21_Click(sender As Object, e As EventArgs) Handles BunifuThinButton21.Click
        cargarTipos()
    End Sub

    Private Sub dtimpuestos_CellContentClick(sender As Object, e As DataGridViewCellEventArgs) Handles dtimpuestos.CellContentClick

    End Sub

    Private Sub dtimpuestos_CellContentDoubleClick(sender As Object, e As DataGridViewCellEventArgs) Handles dtimpuestos.CellContentDoubleClick
        DocumentosNecesarios.idTipo = dtimpuestos.Rows(dtimpuestos.CurrentRow.Index).Cells(0).Value
        DocumentosNecesarios.Show()

    End Sub
End Class

File: Ent_UsAdmin/Creditos/CatTiposdeCredito.cs
using System;
using System.Data.SqlClient;
using System.Windows.Forms;
using Microsoft.VisualBasic.CompilerServices;

namespace ConfiaAdmin
{

    public partial class CatTiposdeCredito
    {
        public CatTiposdeCredito()
        {
            InitializeComponent();
        }
        private void CatTiposdeCredito_Load(object sender, EventArgs e)
        {
            cargarTipos();
        }
        public void cargarTipos()
        {
            dtimpuestos.Rows.Clear();
            try
            {
                string strimpuestos;
                Module1.iniciarconexionempresa();

                strimpuestos = "select * from TiposDeCredito";

                var ejec = new SqlCommand(strimpuestos);
                ejec.Connection = Module1.conexionempresa;
                string id, nombre, Modalidad, Tipo, Plazo;

                var myreaderimpuestos = ejec.ExecuteReader();
                while (myreaderimpuestos.Read())
                {
                    id = Conversions.ToString(myreaderimpuestos["id"]);
                    nombre = Conversions.ToString(myreaderimpuestos["nombre"]);
                    Modalidad = Conversions.ToString(myreaderimpuestos["Modalidad"]);
                    Tipo = Conversions.ToString(myreaderimpuestos["Tipo"]);
                    Plazo = Conversions.ToString(myreaderimpuestos["Plazo"]);
                    dtimpuestos.Rows.Add(id, nombre, Modalidad, Tipo, Plazo);

                }
                myreaderimpuestos.Close();
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message);
            }

        }

        private void btn_agregar_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.AgregarTipoCredito.Show();
        }

        private void BunifuThinButton21_Click(object sender, EventArgs e)
        {
            cargarTipos();
        }

        private void dtimpuestos_CellContentClick(object sender, DataGridViewCellEventArgs e)
        {

        }

        private void dtimpuestos_CellContentDoubleClick(object sender, DataGridViewCellEventArgs e)
        {
            My.MyProject.Forms.DocumentosNecesarios.idTipo = Conversions.ToInteger(dtimpuestos.Rows[dtimpuestos.CurrentRow.Index].Cells[0].Value);
            My.MyProject.Forms.DocumentosNecesarios.Show();

        }
    }
}